data/mappings.tsv

```diff
@@ -773,7 +773,7 @@ mim	is_phenotype	date_fetched	umls_ids	orphanet_ids
 118510	False	2025-03-25	C1332758	
 118511	False	2025-03-20	C1413406	
 118600	True	2025-03-19	C0856830	1416
-118610	False	2025-03-25	C1861580	
+118610	False	2025-04-27	C1861580	
```

Comment:
Only 187 lines in the diff(s). This is more like it!

However, I can now see a new problem I will have to address:
- #219

Reply:
@twhetzel Just pinging you to get your quick eye on this before I merge